client/src/components/Tea/TeaList/TeaList.jsx

```diff
@@ -1,29 +1,141 @@
 import React from "react";
 import { Link } from "react-router-dom";
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faAngleDown, faAngleUp } from "@fortawesome/free-solid-svg-icons";
 
-export class TeaList extends React.Component {
-  handleDeleteClick = tea => {
-    this.props.handleDelete(tea);
-  };
+library.add(faAngleDown, faAngleUp);
 
+export class TeaList extends React.Component {
   render() {
+    const handleSortClick = this.props.handleSortClick;
+    const sortColumn = this.props.sortColumn;
+    const sortOrder = this.props.sortOrder;
+
     return (
       <div className="container">
         <table className="table is-striped is-fullwidth">
           <thead>
             <tr>
-              <td>Name</td>
-              <td>Brand</td>
-              <td>Type</td>
-              <td>Quantity</td>
+              <td>
```

Comment:
All of your `td`s look identical, with the exception of the text they display and the name attribute of the sort column.  Seems like a good indication that a component or function is in order, perhaps TableColumnHeader.  If you don't want the whole `td` to be a component, that's fine, but something to DRY up that `button` and `span` boilerplate would be good, e.g.
```
  headerSortElements(attr) {
    return (
      <React.Fragment>
        <button
          className="button is-small"
          type="button"
          aria-pressed="false"
          onClick={() =>
            this.props.handleSortClick(
              attr,
              this.props.sortColumn === attr && this.props.sortOrder === "asc"
                ? "desc"
                : "asc"
            )
          }
        >
          <span className="icon">
            <i className="fas">
              <FontAwesomeIcon
                icon={
                  this.props.sortColumn === attr && this.props.sortOrder === "asc"
                    ? "angle-up"
                    : "angle-down"
                }
              />
            </i>
          </span>
        </button>
      </React.Fragment>
    )
  }
```
Because this is only used in this component, I think it would be fine for the function to live within your `TeaList` class.